data/InputData.encoding.tim.odetimpayload.xml

```diff
@@ -1 +1 @@
uTimeout><rsuPassword>password</rsuPassword></rsus><rsus><rsuTarget>127.0.0.3</rsuTarget><rsuUsername>v3user</rsuUsername><rsuRetries>1</rsuRetries><rsuTimeout>1000</rsuTimeout><rsuPassword>password</rsuPassword></rsus><snmp><mode>1</mode><deliverystop>2018-12-01T17:47:11-05:15</deliverystop><rsuid>0083</rsuid><deliverystart>2017-12-01T17:47:11-05:00</deliverystart><enable>1</enable><channel>178</channel><msgid>31</msgid><interval>1</interval><status>4</status></snmp></request><payloadType>us.dot.its.jpo.ode.model.OdeTimPayload</payloadType><serialId><streamId>cfaa63fb-e3f8-4c01-9516-648a95bd1cbe</streamId><bundleSize>1</bundleSize><bundleId>2</bundleId><recordId>2</recordId><serialNumber>0</serialNumber></serialId><receivedAt>2017-10-04T13:41:47.862Z[UTC]</receivedAt><schemaVersion>2</schemaVersion><generatedAt>2017-08-10T21:02:14.799Z[UTC]</generatedAt><logFileName>TimLogDuringEvent_1504882236_2001_3A470_3A41af_3A1_3A0_3A63ff_3Afead_3A525.csv</logFileName><validSignature>true</validSignature><sanitized>false</sanitized><encodings><encodings><elementName>root</elementName><elementType>MessageFrame</elementType><encodingRule>UPER</encodingRule></encodings></encodings></metadata><payload><dataType>us.dot.its.jpo.ode.model.OdeHexByteArray</dataType><data><MessageFrame><messageId>31</messageId><value><TravelerInformation><msgCnt>1</msgCnt><timeStamp>309505</timeStamp><packetID>000000000000000000</packetID><urlB>null</urlB><dataFrames><TravelerDataFrame><notUsed>0</notUsed><frameType><advisory/></frameType><msgId><roadSignID><position><lat>416784730</lat><long>-1087827750</long><elevation>9171</elevation></position><viewAngle>0101010101010100</viewAngle><mutcdCode><guide/></mutcdCode><crc>0000</crc></roadSignID></msgId><startYear>2017</startYear><startTime>308065</startTime><durationTime>1</durationTime><priority>0</priority><notUsed1>0</notUsed1><regions><GeographicalPath><name>Testing TIM</name><id><region>0</region><id>33</id></id><anchor><lat>412500807</lat><long>-1110093847</long><elevation>20206</elevation></anchor><laneWidth>700</laneWidth><directionality><both/></directionality><closedPath><false/></closedPath><direction>0000000000010100</direction><description><path><scale>0</scale><offset><ll><nodes><NodeLL><delta><node-LL3><lon>14506</lon><lat>31024</lat></node-LL3></delta></NodeLL><NodeLL><delta><node-LL3><lon>14568</lon><lat>30974</lat></node-LL3></delta></NodeLL><NodeLL><delta><node-LL3><lon>14559</lon><lat>30983</lat></node-LL3></delta></NodeLL><NodeLL><delta><node-LL3><lon>14563</lon><lat>30980</lat></node-LL3></delta></NodeLL><NodeLL><delta><node-LL3><lon>14562</lon><lat>30982</lat></node-LL3></delta></NodeLL></nodes></ll></offset></path></description></GeographicalPath></regions><notUsed2>0</notUsed2><notUsed3>0</notUsed3><content><advisory><SEQUENCE><item><itis>513</itis></item></SEQUENCE></advisory></content><url>null</url></TravelerDataFrame></dataFrames></TravelerInformation></value></MessageFrame></data></payload></OdeAsn1Data>
```

Comment:
It would be helpful if we could update the README.md where each .xml in this data 
 folder generated/output to?